scripts/commands/checkmission.lua

```diff
@@ -13,7 +13,20 @@ cmdprops =
 
 function onTrigger(player,logId,target)
     
+    local logName;
     logId = tonumber(logId) or _G[logId];
+    if ((type(logId) == "table")) then
+        logName = logId.full_name;
+        logId = logId.mission_log;
+    else
+        if (logId <= 2) then
+            local logNames = {"San d'Oria", "Bastok", "Windurst"}
```

Comment:
well if it didn't ride in with yer changes, must have been @takhlaq DEMOLISH! 😄 I volunteer you to see wtf is up and do whatever is needed, coz I know it was you that went _G happy with commands.